### DISCUSSION TITLE
配置问题

### DISCUSSION THREAD

**Discussion by lxlx2084** [NovaSearch-Team/RAG-Retrieval#41] [Q&A] [answered]
我要微调reranker，你的介绍里也说了可以调bge-reranker-large，train_reranker.sh文件里根本没有这个接口啊？我用的第一个，提示bge-reranker-large没有bertlayer类

#model_bert,fsdp(ddp)
# CUDA_VISIBLE_DEVICES="1"  nohup  accelerate launch --config_file ../../../config/default_fsdp.yaml train_reranker.py  \
#--model_name_or_path "/data/liuxiang/58/10_llm/6_比赛/bge-reranker-large" \
#--dataset "../../../example_data/t2rank_100.jsonl" \
#--output_dir "./output/t2ranking_100_example" \
#--model_type "cross_encoder" \
#--loss_type "classfication" \
#--batch_size 32 \
#--lr 5e-5 \
#--epochs 2 \
#--num_labels 1 \
#--log_with  'wandb' \
#--save_on_epoch_end 1 \
#--warmup_proportion 0.1 \
#--gradient_accumulation_steps 3 \
#--max_len 512 \
# >./logs/t2ranking_100_example.log &

#model_bert,fsdp(ddp)
# CUDA_VISIBLE_DEVICES="0"  nohup  accelerate launch --config_file ../../../config/default_fsdp.yaml train_reranker.py  \
#--model_name_or_path "hfl/chinese-roberta-wwm-ext" \
#--dataset "../../../example_data/t2rank_100.jsonl" \
#--output_dir "./output/t2ranking_100_example" \
#--model_type "cross_encoder" \
#--loss_type "classfication" \
#--batch_size 32 \
#--lr 5e-5 \
#--epochs 2 \
#--num_labels 1 \
#--log_with  'wandb' \
#--save_on_epoch_end 1 \
#--warmup_proportion 0.1 \
#--gradient_accumulation_steps 3 \
#--max_len 512 \
# >./logs/t2ranking_100_example.log &

**Comment by NLPJCL** [comment, marked as answer]
https://github.com/NLPJCL/RAG-Retrieval/tree/master/rag_retrieval/train/reranker#:~:text=xlmroberta_default_config%2C%20%E5%A6%82%E6%9E%9C%E8%A6%81%E5%9C%A8bge%2Dreranker%2Dbase%E5%92%8Cbce%2Dreranker%2Dbase_v1%E7%9A%84%E5%9F%BA%E7%A1%80%E4%B8%8A%E8%BF%9B%E8%A1%8C%E5%BE%AE%E8%B0%83%EF%BC%8C%E9%87%87%E7%94%A8%E8%AF%A5%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6%EF%BC%8C%E5%9B%A0%E4%B8%BA%E4%B8%A4%E8%80%85%E5%9C%A8%E5%A4%9A%E8%AF%AD%E8%A8%80%E7%9A%84xlmroberta%E7%9A%84%E5%9F%BA%E7%A1%80%E4%B8%8A%E8%AE%AD%E7%BB%83%E8%80%8C%E6%9D%A5%E3%80%82

**Comment by NLPJCL** [comment]
后面有问题，请在issue区提问，谢谢。

**Reply by lxlx2084** [reply]
好的，感谢
